pricing.js

```diff
@@ -1,37 +1,45 @@
+function formatPrice(price) {
+  return parseInt(price * 100) / 100
+}
+
+function calculateEmployerContribution(product, price, dollarsOff) {
```

Comment:
Good job breaking this up into a separate function, previously both calculate functions were doing this and as we know we want our code to be DRY (don't repeat yourself). Moving it into a separate function and calling it in both places means we only have one place the code lives.

Unfortunately this function currently doesn't work as we'd need it to. You are correctly calculating the discounted price but not actually returning it. This means the DiscountedPrice on lines on lines 35 and 39 would actually be undefined.

This is why the tests are coming back with NaN.
![image](https://user-images.githubusercontent.com/5775716/65338926-31977700-db99-11e9-9e9f-2941f375f292.png)